.github/workflows/build-containers.yml

```diff
@@ -377,6 +377,17 @@ jobs:
           sbom: true
           provenance: true
 
+      - name: Build single-platform image for Trivy scan
+        id: build-for-scan
+        uses: docker/build-push-action@263435318d21b8e681c14492fe198d362a7d2c83 # v6
+        with:
+          context: ${{ steps.dockerfile.outputs.dockerfile_dir }}
+          file: ${{ steps.dockerfile.outputs.dockerfile_path }}
+          platforms: linux/amd64
```

Comment:
nit: should we build for arm (I think the github runners are arm based so it might be faster?)